### PR TITLE
InlineImage

### PR DESCRIPTION
**Pull request by klopfdreh**

### CODE REVIEW

File: wicket-core/src/test/java/org/apache/wicket/markup/html/image/InlineImageTest.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+
+import org.apache.wicket.util.tester.WicketTester;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+
+public class InlineImageTest
```
Comment (martin-g) [inline wicket-core/src/test/java/org/apache/wicket/markup/html/image/InlineImageTest.java]: extend from WicketTestCase and reuse its before/after methods.

Comment (klopfdreh) [inline wicket-core/src/test/java/org/apache/wicket/markup/html/image/InlineImageTest.java]: No, you said this some time ago! :-S - I am going to change it, now. Thanks!

**Comment by martin-g** [comment]
:+1:

**Comment by klopfdreh** [comment]
Thanks for the fast review! :smiley:

**Pull request merged**